tools/integration_tests/util/operations/dir_operations.go

```diff
@@ -68,9 +69,18 @@ func CreateDirectoryWithNFiles(numberOfFiles int, dirPath string, prefix string,
 		// Create file with name prefix + i
 		// e.g. If prefix = temp  then temp1, temp2
 		filePath := path.Join(dirPath, prefix+strconv.Itoa(i))
-		_, err := os.Create(filePath)
+		file, err := os.Create(filePath)
 		if err != nil {
 			t.Errorf("Create file at %q: %v", dirPath, err)
 		}
+
+		// Closing file at the end.
+		defer CloseFile(file)
+	}
+}
+
+func RemoveDir(dirPath string) {
```

Comment:
can we put all these methods in one file. its difficult to find for someone new to find what methods exist

Reply:
It is in one file right. All the directory operations functions operations/dir_operations are in one place and all the file operations are in one place operations/file_operations